### DISCUSSION TITLE
OverlayPanel in fixed div scrolls with body out of view.

### DISCUSSION THREAD

**Discussion by graficon** [primefaces/community#1705] [PrimeNG]
In a typical (top) header navigation bar we have an overlay panel to show user options such as logout, etc.
The nav header div has position 'fixed'.

When scrolling the document page, the header nav bar stays fixed as expected, but the overlay panel (when it's open) scrolls out of view. The link (button) on the header bar that opens the overlay panel is not clickable any more (after panel has scrolled out of view).

Is there a way to retain the panel's position on the nav bar so that it does not scroll out of view when you scroll the document body?

Here is a stackblitz: https://stackblitz.com/edit/qtacf6?file=src%2Fapp%2Fdemo%2Foverlay-panel-basic-demo.html,src%2Fapp%2Fdemo%2Foverlay-panel-basic-demo.ts

Thanks in advance

**Comment by cetincakiroglu** [comment]
Hi,

You need to append the overlay to the element inside the navbar. Here is a simple example how to do it: https://stackblitz.com/edit/qtacf6?file=src%2Fapp%2Fdemo%2Foverlay-panel-basic-demo.html,src%2Fapp%2Fdemo%2Foverlay-panel-basic-demo.ts

**Reply by graficon** [reply]
Hi,
I don't see any differences between my example and yours.  The overlay panel still scrolls out of view and does not stay attached (and visible) to the fixed header.

Thanks for looking into this.
